src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -289,6 +268,10 @@ protected override bool ReleaseHandle()
         private SafeSslHandle() : base(IntPtr.Zero, true)
         {
         }
+
+        internal SafeSslHandle(IntPtr validSslPointer, bool ownsHandle) : base(validSslPointer, ownsHandle)
```

Comment:
You pass IntPtr.Zero to the base ctor, and in your ctor you set this.handle to your value or call SetHandle with your value.